src/benchmarks/gc/src/exec/GCPerfSim/GCPerfSim.cs

```diff
@@ -1725,11 +1853,13 @@ struct OldArr
 {
     public readonly ITypeWithPayload?[] items;
     public ulong TotalLiveBytes { get; private set; }
+    public uint NonEmptyLength;
```

Comment:
why is the var that indicates the # of elements in the array is called NonEmptyLength?